src/components/cesium_map/api/spatial.js

```diff
@@ -303,18 +303,32 @@ export class ISamplesSpatial {
     });
 
     this.viewer = new Cesium.Viewer(element, {
-      infoBox:false,
+      // infoBox:false,
```

Comment:
If we aren't using this, please remove this line rather than checking it in commented out (unless you plan on resurrecting it shortly)